homework/07LessonSeven/script.js

```diff
@@ -69,20 +69,25 @@ $(document).on('ready', function() {
   // Problem 9:
   // select the element with id="insert-text"
   // then set the text of this element to "Text Has Been Inserted"
-
+  $('#insert-text').text("Text Has Been Inserted");
 
   // Problem 10:
   // select the element with id="add-attribute"
   // add the attribute data-color="red" to that element
-
+  window.addAttribute = $("#add-attribute");
+  window.addAttribute[0].setAttribute("data-color", "red");
```

Comment:
This exercise calls for jQuery selectors and methods. 

Condense your code by using the .attr() method instead of setAttribute for problem Problem 10.